### PR TITLE
Mobile responsive portfolio project

### PR DESCRIPTION
**Pull request by lilyfrancis**
This is a responsive portfolio project by microverse to test the use of Flexbox and Positioning. In this Project I added the following:

- Setup Linters
- Index File
- A CSS style file
- A ReadMe File

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,63 +1,58 @@
...
+> This is a responsive portfolio project by microverse to test the use of Flexbox for Positioning and also grids, images and Background, 
+
+![screenshot](images/screenshot.png)
+
+
+## Built With
+
+- Html
+- CSS
+
```
Comment (SajjadAhmad14) [inline README.md:12]: - [ ] Please align logo and nav elements vertically in the navigation bar.  You can spot misalignment in the below screenshot. Kindly use font colors for navigation elements from the design only.
nav

File: README.md [hunk 2]
```diff
@@ -1,63 +1,58 @@
...
+
```
Comment (SajjadAhmad14) [inline README.md:13]: Please align social icons vertically. Kindly see the below screenshot for reference.
icons

File: README.md [hunk 3]
```diff
@@ -1,63 +1,58 @@
...
+> This is a responsive portfolio project by microverse to test the use of Flexbox for Positioning and also grids, images and Background, 
+
+![screenshot](images/screenshot.png)
+
+
+## Built With
+
+- Html
+- CSS
```
Comment (SajjadAhmad14) [inline README.md:11]: Mobile view does not show background images for header and sections. Kindly add background images from the design. Moreover, background images for tablet view are not the same as in the original design and on the desktop view, these pictures are not positioned the same as in the design.
back

File: README.md [hunk 4]
```diff
@@ -1,63 +1,58 @@
...
+# My Portfolio
+
+> This is a responsive portfolio project by microverse to test the use of Flexbox for Positioning and also grids, images and Background, 
+
+![screenshot](images/screenshot.png)
+
+
+## Built With
+
+- Html
```
Comment (SajjadAhmad14) [inline README.md:10]: Please align the picture and text in the portfolio section. Moreover, a button is overlapping text on mobile and tablet views. Please see the below screenshot and fix the issues.
project1

File: README.md [hunk 5]
```diff
@@ -1,63 +1,58 @@
...
-This project is [MIT](./MIT.md) licensed.
+# My Portfolio
+
+> This is a responsive portfolio project by microverse to test the use of Flexbox for Positioning and also grids, images and Background, 
+
+![screenshot](images/screenshot.png)
+
+
+## Built With
+
```
Comment (SajjadAhmad14) [inline README.md:9]: Please align buttons with the above text section horizontally. See the 
project2
below screenshot.

Comment (SajjadAhmad14) [inline README.md:9]: Please align buttons with the above text and also align the text section with pictures and change background color of `See Project` button. Kindly see the below screenshot.
project3

Comment (SajjadAhmad14) [inline README.md:9]: Please align picture with text section and align buttons horizontally based on text their above,
project4

File: README.md [hunk 6]
```diff
@@ -1,63 +1,58 @@
...
-
-This project is [MIT](./MIT.md) licensed.
+# My Portfolio
+
+> This is a responsive portfolio project by microverse to test the use of Flexbox for Positioning and also grids, images and Background, 
+
+![screenshot](images/screenshot.png)
+
+
+## Built With
```
Comment (SajjadAhmad14) [inline README.md:8]: Icons in the about me section are not aligned and scattered. Moreover, social icons are not aligned horizontally with text above them. Please see the 
about
below screenshot.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Lilian Francis, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
- [ ] README is professional.
- [ ] GitHub flow is followed properly.
## To Improve 🙏

- [ ] Please check inline comments under the review
- [ ] Please align buttons, navigation items,  text sections, and images of projects horizontally and vertically.
- [ ] Please stick to the design. Kindly use background images and use the same font colors as defined in the design.
## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html [hunk 7]
```diff
@@ -1,315 +1,357 @@
...
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous"/>
+    <link rel="stylesheet" href="style.css" />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <header id="header">
```
Comment (SajjadAhmad14) [inline index.html:13]: Logo and navigation items are still not aligned vertically. Please align them vertically.

File: index.html [hunk 8]
```diff
@@ -1,315 +1,357 @@
...
-    </div>
-    <div id="cssmenu">
-
-      <ul>
-        <li><a href="#">Contact</a></li>
-        <li><a href="#">About</a></li>
-        <li><a href="#">Potfolio</a></li>
-      </ul>
-    </div>
-    <div class="about-flex-container">
```
Comment (SajjadAhmad14) [inline index.html:34]: Thank you for adding background images on mobile view. But they are not the same as in the design and their position is also not matching with the design. Please see the design page for mobile and make corrections accordingly. Moreover, an image is overlapping icons on the large view. Kindly see the below screenshot for reference.
overlap

File: index.html [hunk 9]
```diff
@@ -1,315 +1,357 @@
...
-  </section>
-  <section class="nature-section">
-    <div class="natureimg">
-
-    </div>
-    <div class="section-grid">
-      <div id="item-0">
-      </div>
-      <div id="item-1">
-        <h3>Tonic</h3>
```
Comment (SajjadAhmad14) [inline index.html:81]: The image for the project should have a full container width on medium viewport. Moreover, tech stack info is also missing. Please see the below screenshot for reference.
project1

File: index.html [hunk 10]
```diff
@@ -1,315 +1,357 @@
...
+          </p>
+          <div class="coverUl">
+            <ul class="contact-form">
+              <li class="contact-item">
+                <input type="text" name="name" placeholder="Yeremias" maxlength="30" required/>
+              </li>
+              <li class="contact-item">
+                <input type="email" name="email" placeholder="natajayanj14@gmai.com" required/>
+              </li>
+              <li class="contact-item">
```
Comment (SajjadAhmad14) [inline index.html:348]: [ OPTIONAL ] Text area section is not the same as the design. Please see the below screenshot and correct it accordingly. 
text-box

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Lilian Francis, 
 ### Thank you for working on changes suggested previously :ok_hand:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by Murodjon000 — APPROVED** [review]
**Hi Lilian Francis 👋,** 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**